tests/fixtures/parsers/typescript-parsers/member-call-expr-with-assertion.js

```diff
@@ -0,0 +1,366 @@
+            loc: {
+                start: {
+                    line: 1,
+                    column: 13,
+                },
+                end: {
+                    line: 1,
+                    column: 22,
+                },
+            },
+        },
+        {
+            type: "Identifier",
+            value: "number",
+            range: [23, 29],
+            loc: {
+                start: {
+                    line: 1,
+                    column: 23,
+                },
+                end: {
+                    line: 1,
+                    column: 29,
+                },
+            },
+        },
+        {
+            type: "Punctuator",
+            value: ")",
+            range: [29, 30],
+            loc: {
+                start: {
+                    line: 1,
+                    column: 29,
+                },
+                end: {
+                    line: 1,
+                    column: 30,
+                },
+            },
+        },
+        {
+            type: "Punctuator",
+            value: ".",
+            range: [30, 31],
+            loc: {
+                start: {
+                    line: 1,
+                    column: 30,
+                },
+                end: {
+                    line: 1,
+                    column: 31,
+                },
+            },
+        },
+        {
+            type: "Identifier",
+            value: "toFixed",
+            range: [31, 38],
+            loc: {
+                start: {
+                    line: 1,
+                    column: 31,
+                },
+                end: {
+                    line: 1,
+                    column: 38,
+                },
+            },
+        },
+        {
+            type: "Punctuator",
+            value: "(",
+            range: [38, 39],
+            loc: {
+                start: {
+                    line: 1,
+                    column: 38,
+                },
+                end: {
+                    line: 1,
+                    column: 39,
+                },
+            },
+        },
+        {
+            type: "Punctuator",
+            value: ")",
+            range: [39, 40],
+            loc: {
+                start: {
+                    line: 1,
+                    column: 39,
+                },
+                end: {
+                    line: 1,
+                    column: 40,
+                },
+            },
+        },
+        {
+            type: "Punctuator",
+            value: ";",
+            range: [40, 41],
+            loc: {
+                start: {
+                    line: 1,
+                    column: 40,
+                },
+                end: {
+                    line: 1,
+                    column: 41,
+                },
+            },
+        },
+    ],
+    loc: {
+        start: {
+            line: 1,
+            column: 0,
+        },
+        end: {
+            line: 2,
+            column: 0,
```

Comment:
```suggestion
            line: 1,
            column: 41,
```

To remove an extra `\n`.